javatests/google/registry/rde/imports/RdeDomainImportActionTest.java

```diff
@@ -174,6 +331,14 @@ private void pushToGcs(ByteSource source) throws IOException {
     return result;
   }
 
+  private static BillingEvent.Recurring loadAutorenewBillingEventForDomain(DomainResource domain) {
```

Comment:
Why not?  It's a helpful annotation in all circumstances.  Here, it'll make it easier to write more tests, since the behavior of these helper methods is better documented.